cellular_configs/v1alpha1/amfconfig_types.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2023 The Nephio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// AmfConfigStatus defines the observed state of AmfConfig
+type AmfConfigStatus struct {
```

Comment:
Convention is to match the case of the entire "initialism" to the case of the first character. So, in struct names "Amf" -> "AMF". In exported Go field names the same. In the JSON names, "amf" if it's the start of the field name, but "AMF" if it appears after the first word.

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions

That is, you need to change "Amf" to either "amf" or "AMF" everywhere, in every file, based upon where it fits in the convention.

Reply:
Okay I will make this change, thanks for the conventions.